syntax = "proto3";

option
java_package = "com.huasheng.quant.open.sdk.protobuf.trade.request.futures";
option java_outer_classname = "FuturesQueryFundInfoRequestProto";
message FuturesQueryFundInfoRequest {
}
